### PR TITLE
PRO - 1007 "Fetching rates" issue during swaps

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
* Points
   1. Fixed Fetching rates issue
   2. Update some conditions in fetching rates

https://user-images.githubusercontent.com/78720269/214473165-001be788-162f-4d8c-869b-2f6da76f8e45.mp4

### CODE REVIEW

File: src/actions/assetsActions.js [hunk 1]
```diff
@@ -615,7 +616,7 @@ export const fetchAssetsBalancesAction = () => {
     dispatch({ type: SET_FETCHING_ASSETS_BALANCES, payload: true });
 
     dispatch(fetchSupportedAssetsAction());
-    dispatch(fetchPopularAssetsAction());
+    if (!popularAssets?.[CHAIN.POLYGON]) dispatch(fetchPopularAssetsAction());
```
Comment (IAmKio) [inline src/actions/assetsActions.js:619]: Was there an issue with Polygon popular assets?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a question (also asked on Slack) but other than this - we're looking good

File: src/actions/assetsActions.js (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/actions/assetsActions.js:619]: No no there is not an issue. I put a condition that if popular assets are available then don’t call this method again. we have not any other way to find assets available or accept this one.

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**